TWLight/applications/views.py

```diff
@@ -1445,27 +1459,29 @@ def dispatch(self, request, *args, **kwargs):
 
         if app.partner.is_not_available:
             return_url = self._set_return_url(self._get_return_url())
-            # Translators: When a user tries to renew their resource, they receive this message if the partner is not available.
             messages.add_message(
                 request,
                 messages.WARNING,
-                _(
-                    "Cannot renew application at this time as partner is not available. "
+                # fmt: off
+                # Translators: When a user tries to renew their resource, they receive this message if the partner is not available.
+                _("Cannot renew application at this time as partner is not available. "
                     "Please check back later, or contact us for more information."
                 ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string